mapbox/libjava-services/src/main/java/com/mapbox/services/api/utils/turf/TurfHelpers.java

```diff
@@ -88,4 +91,41 @@ public static double distanceToRadians(double distance, String units) throws Tur
     }
     return distance / factor;
   }
+
+  /**
+   * Converts a distance to the requested unit.
+   * Valid units: miles, nauticalmiles, inches, yards, meters, metres, kilometers, centimeters, feet
+   *
+   * @param distance     the distance to be converted
+   * @param originalUnit of the distance
+   * @return the converted distance
+   * @since 2.2.0
+   */
+  public static double convertDistance(double distance, String originalUnit) {
+    return convertDistance(distance, originalUnit, TurfConstants.UNIT_DEFAULT);
+  }
+
+  /**
+   * Converts a distance to the requested unit.
+   * Valid units: miles, nauticalmiles, inches, yards, meters, metres, kilometers, centimeters, feet
+   *
+   * @param distance     the distance to be converted
+   * @param originalUnit of the distance
+   * @param finalUnit    returned unit, {@link TurfConstants#UNIT_DEFAULT} if not specified.
+   * @return the converted distance
+   * @since 2.2.0
+   */
+  public static double convertDistance(double distance, String originalUnit, String finalUnit) {
+    Double factor = factors.get(originalUnit);
+    if (factor == null) {
+      throw new TurfException("Invalid unit.");
+    } else if (!(distance >= 0)) {
+      throw new TurfException("distance must be a positive number");
```

Comment:
Please change the message to `Distance must be a positive number.` for consistency.